LayoutTests/fast/dom/no-scroll-when-command-click-fragment-URL.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>Fragment Navigation - ensure opening a fragment URL does not navigate on activating page.</title>
+<meta name="assert" content="This test checks command-clicking on a fragment URL does not navigate on the current page.">
+
+<head>
+<script src="../../resources/ui-helper.js"></script>
+
+<script>
+if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+}
+
+  
+ async function runTest()
+ {
+    if (!testRunner.runUIScript)
+        return;
```

Comment:
Nit - the indentation is a bit strange in this file (there's a mix of 2, 3, and 4-space indents)